src/multi_vector_simulator/F2_autoreport.py

```diff
@@ -956,9 +963,9 @@ def create_app(results_json, path_sim_output=None):
                                 ),
                             ),
                             insert_body_text(
-                                "This results in the following KPI of the dispatch:"
+                                "This results in the following KPI of the dispatch per energy sector:"
                             ),
-                            # TODO the table with renewable share, emissions, total renewable generation, etc.
+                            make_dash_data_table(df_kpi_sectors),
```

Comment:
I would move the table to the end of the report, below `Energy System: Key Performance Indicators (KPIs)` and the energy system wide KPI. 

However, it does not really matter... the order of the report is a mess and should probably be changed. Probably 

1.  Project and economic data
2. Energy system plot 
3. KPI 
4. Costs 
5. Dispatch 
6. Assumptions (including resources, demands...) 
 
would make more sense...

Reply:
I would say, whoever wants to reschuffle the order of the report can do it now in a subsequent PR :)